code/game/gamemodes/blob/blobs/shield.dm

```diff
@@ -5,7 +5,7 @@
 	desc = "A solid wall of slightly twitching tendrils."
 	health = 150
 	maxhealth = 150
-	brute_resist = 0.1
+	brute_resist = 0.25

```

Comment:
Reduces ability to soak damage by 250%, is only twice as defensive as just a normal blob tile.

Feels really intense on this nerf.